src/Microsoft.Azure.ServiceBus/OnMessageOptions.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus
+{
+    using System;
+    using Primitives;
+
+    public sealed class OnMessageOptions
+    {
+        int maxConcurrentCalls;
+        TimeSpan autoRenewTimeout;
+
+        /// <summary>Initializes a new instance of the <see cref="OnMessageOptions" /> class.</summary>
```

Comment:
It would be good for the code to document the defaults.
Otherwise, it's hoping that msdn / doco will get updated, which is always a PITA.